src/Header.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components/macro';
+
+export default function Header() {
+    return (
+        <Container>
+            <h1>onemoreminiround</h1>
+            <Button class="menu-button">
+                <svg
+                    class="icon"
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                >
+                    <path
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        stroke-width="2"
+                        d="M4 6h16M4 12h16M4 18h16"
+                    />
+                </svg>
```

Comment:
I am considering to just import this, also I just saw a `class` 😱

Reply:
:O :O :O

Reply:
Imma fix this later